### PR TITLE
Make - install hashing headers, used in MFEM

### PR DESCRIPTION
**Pull request by jeremylt**

### CODE REVIEW

**Review by nbeams — APPROVED** [review]
LGTM

File: include/ceed-hash.h [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #define _ceed_hash_h
 
 #include <ceed.h>
-#include <khash.h>
+#include <ceed-khash.h>
```
Comment (jedbrown) [inline include/ceed-hash.h:23]: Renaming the file isn't enough to enable a user to include their own `khash.h` separately. Could you update the header guard in `ceed-khash.h`?

**Review by jedbrown — APPROVED** [review]

File: include/ceed-hash.h (hunk 1 above)
Comment (jeremylt) [reply include/ceed-hash.h:23]: updated

**Comment by jeremylt** [comment]
It looks like rocm 3.8 + libCEED is broken?

**Comment by nbeams** [comment]
Unfortunately, I don't currently have access to a machine with ROCm 3.8 (only 3.7), so I can't test it.  From the output of the failure, it seems like something's going wrong with `hipconfig`?

**Comment by jedbrown** [comment]
I was just in the midst of upgrading/fixing a recurring packaging issue and rebooting. It passed now.

**Pull request merged**